hello_world/hello_world.py

```diff
@@ -0,0 +1,5 @@
+"""
+To run:
```

Comment:
We should make sure we use the same wording in each of the files
```suggestion
Run with:
```

Reply:
> We should make sure we use the same wording in each of the files
> ```suggestion
> Run with:
> ```

I've recorded a reply with VCR:

[17s - 20s](http://localhost:8000/video/video-code-review/hello-world/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjoxfQ.Go2yV3jw_tyxS2ur5U0g2FCZtoc/?start=17&end=20): We should make sure we use the same wording in each of the files - ```su...

Reply:
> We should make sure we use the same wording in each of the files
> ```suggestion
> Run with:
> ```

I've recorded a reply with VCR:

[8s - 13s](http://localhost:8000/video/video-code-review/hello-world/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjoyfQ.nWO8GH6XKJG-OSuT_4qvbroStds/?start=8&end=13): We should make sure we use the same wording in each of the files - ```su...

Reply:
> We should make sure we use the same wording in each of the files
> ```suggestion
> Run with:
> ```

I've recorded a reply with VCR:

[10s - 15s](http://localhost:8000/video/video-code-review/hello-world/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjozfQ.XhFOBoQYAly3U_HHVGmSzKg3TWY/?start=10&end=15): We should make sure we use the same wording in each of the files - ```su...

Reply:
> We should make sure we use the same wording in each of the files
> ```suggestion
> Run with:
> ```

I've recorded a reply with VCR:

[12s - 23s](http://localhost:8000/video/video-code-review/hello-world/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjo0fQ.TJPC78dsBS_HEuBmyhoeW8Vdb-8/?start=12&end=23): We should make sure we use the same wording in each of the files - ```su...

Reply:
> We should make sure we use the same wording in each of the files
> ```suggestion
> Run with:
> ```

I've recorded a reply with VCR:

[4s - 6s](http://localhost:8000/video/video-code-review/hello-world/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjo1fQ.aH60pVvIgGSBHlk4xXffHcEUtmk/?start=4&end=6): We should make sure we use the same wording in each of the files - ```su...

Reply:
> We should make sure we use the same wording in each of the files
> ```suggestion
> Run with:
> ```

I've recorded a reply with VCR:

[9s - 16s](http://localhost:8000/video/video-code-review/hello-world/1/eyJ1c2VyIjoicHllcHllIiwibnVtYmVyIjo2fQ.d_No1HMSSGw6FMeUe8QCfoMzC3Y/?start=9&end=16): We should make sure we use the same wording in each of the files - ```su...

Reply:
> We should make sure we use the same wording in each of the files
> ```suggestion
> Run with:
> ```

I've recorded a video reply:

[00:41 - 00:54](https://app.videocode.review/video/video-code-review/hello-world/1/WyJkYXZpZG1vc3MiLCJtcDQiLDE2MTMwMzczNDZd.pKE-kaOviQXI-mBP8VKQlOTYYvY/?start=41&end=54): We should make sure we use the same wording in each of the files - ```su...